lib/heapwatch.js

```diff
@@ -21,28 +21,31 @@ class HeapWatch {
         this.timeoutHandle = undefined;
         this.gcReportInterval = undefined;
         this.cumulativeGCTimes = Object.assign({}, ZERO_CUMULATIVE_GC_INTERVAL);
+        this.reportStatsHandler = (stats) => {
+                // Report GC timings to statsd (in nanoseconds).
+                const type = this.gcTypeName(stats.gctype);
+                if (type !== 'unknown') {
+                    this.cumulativeGCTimes[this.gcTypeName(stats.gctype)] += stats.pause;
+                }
+        };
+        this.gcStats = null;
+    }
+
+    gcTypeName(typeID) {
```

Comment:
This is a private method. Prepend with `_`.

Reply:
Done